nemo_text_processing/inverse_text_normalization/mr/data/date/dates.tsv

```diff
@@ -0,0 +1,31 @@
+१	एक
+२	दोन
+३	तीन
+४	चार
+५	पाच
+६	सहा
+७	सात
+८	आठ
+९	नऊ
+१०	दहा
+११	अकरा
+१२	बारा
+१३	तेरा
+१४	चौदा
+१५	पंधरा
+१६	सोळा
+१७	सतरा
+१८	अठरा
+१९	एकोणीस
+२०	वीस
+२१	एकवीस
+२२	बावीस
+२३	तेवीस
+२४	चोवीस
+२५	पंचवीस
+२६	सहावीस
```

Comment:
```suggestion
२६	सव्वीस 
```

Reply:
Thank you for the review. Have changed all the files with the errors.